openmetrics/datadog_checks/openmetrics/data/conf.yaml.example

```diff
@@ -670,3 +670,8 @@ instances:
     #   - <INCLUDE_REGEX>
     #   exclude:
     #   - <EXCLUDE_REGEX>
+
```

Comment:
File `conf.yaml.example` is not in sync, run "ddev validate config openmetrics -s"